client/js/photos.js

```diff
@@ -0,0 +1,54 @@
+document.addEventListener("DOMContentLoaded", () => {
+    loadPhotos();
+    const form = document.querySelector("#formy-mcformface")
+    form.addEventListener("submit", addPhoto)
+});
+
+
+const loadPhotos = async () => {
+    let params = (new URL(document.location)).searchParams;
+    let album = params.get("album");
+    let response = await axios.get(`http://localhost:11000/photos/albums/${parseInt(album)}`);
+    let existingPhotos = response.data.body
```

Comment:
if the album is empty, the browser breaks. you should put this part of the function (lines 12-27) into a condition: if (response.data.body) {}
response.data.body will be null if there are no photos in the album

Reply:
@DouglasMacKrell